spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/RoiAlign.scala

```diff
@@ -354,16 +355,16 @@ class RoiAlign[T: ClassTag] (
 
     for (n <- 0 until num_rois) {
       val index_n = n * channels * pooledW * pooledH
-      var offset_rois = n * roi_cols
+      val offset_rois = n * roi_cols
       val roi_batch_ind = 0
-
-      val roi_start_w = roisDouble(offset_rois) * spatialScale.toDouble
-      val roi_start_h = roisDouble(offset_rois + 1) * spatialScale.toDouble
-      val roi_end_w = roisDouble(offset_rois + 2) * spatialScale.toDouble
-      val roi_end_h = roisDouble(offset_rois + 3) * spatialScale.toDouble
-
-      val roi_width = Math.max(roi_end_w - roi_start_w, 1.0)
-      val roi_height = Math.max(roi_end_h - roi_start_h, 1.0)
+      val alignedOffset = if (aligned) 0.5f else 0.0f
```

Comment:
What does alignedOffser do?

Reply:
Just a offset need to subtract when computing roi border

Reply:
LGTM